.rubocop.yml

```diff
@@ -1,284 +1,23 @@
 require:
   - rubocop/rspec/focused
 
-# This (http://c2.com/cgi/wiki?AbcMetric) is super obnoxious
-AbcSize:
-  Enabled: false
-
-AccessorMethodName:
-  Enabled: false
-
-Alias:
-  Enabled: false
-
-AllCops:
-  Exclude:
-    - !ruby/regexp /vendor\/(?!panoramaed).*$/
-    - "spec/dummy/**/*"
-    - "db/schema.rb"
-    - "db/migrate/**/*"
-  TargetRubyVersion: 2.3
-
-AmbiguousOperator:
-  Enabled: false
-
-AmbiguousRegexpLiteral:
-  Enabled: false
-
-ArrayJoin:
-  Enabled: false
-
-AsciiComments:
-  Enabled: false
-
-AsciiIdentifiers:
-  Enabled: false
-
-AssignmentInCondition:
-  Enabled: true
-
-Attr:
-  Enabled: false
-
-BlockNesting:
-  Enabled: false
-
-BracesAroundHashParameters:
-  Enabled: false
-
-CaseEquality:
-  Enabled: false
-
-CharacterLiteral:
-  Enabled: false
-
-ClassLength:
-  Enabled: false
-
-ClassVars:
-  Enabled: false
-
-CollectionMethods:
-  PreferredMethods:
-    find: detect
-    reduce: inject
-    collect: map
-    find_all: select
-
-ColonMethodCall:
-  Enabled: false
-
-CommentAnnotation:
-  Enabled: false
-
-ConditionPosition:
-  Enabled: false
-
-CyclomaticComplexity:
-  Enabled: false
-
-Debugger:
-  Enabled: true
-  Severity: error
-
-Delegate:
-  Enabled: false
-
-DeprecatedClassMethods:
-  Enabled: false
-
-Documentation:
-  Enabled: false
-
-DotPosition:
+Layout/DotPosition:
```

Comment:
I had to make changes because RuboCop had renamed some rules, so I took the opportunity to just remove the cruft since most of our rules are the same as the defaults.